src/components/CommentList/CommentList.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Comment } from '../Comment';
+import './CommentList.scss';
+
+export const CommentList = ({ comments }) => {
+  const { id } = comments;
+
+  return (
+    <ul className="comment__list">
+      {comments.map(comment => (
+        <li key={id}>
+          <Comment {...comment} />
+        </li>
+      ))}
+    </ul>
+  );
```

Comment:
One more time. `comments` is an array, not object.

```suggestion
  return (
    <ul className="comment__list">
      {comments.map(comment => (
        <li key={comment.id}>
          <Comment {...comment} />
        </li>
      ))}
    </ul>
  );
```